### ISSUE TITLE
z

### ISSUE THREAD

**Issue by TechDudie** [Hyclone/Hyclone#3]

**Comment by TechDudie** [comment]
sorry for originally naming it z, typo
Skyblock and bedwars are 2 great games on hypixel
skyblock shouldn’t be with minions and custom stuff, it should just be a skyblock run like any other minecraft skyblock server, with vote crates, and keys, and money, and /shop for example

**Comment by AFCMS** [comment]
Yes, these gamemodes are planned. (bedwars, skywars, and other minigames have the highest priority)
